Update README.md (#2210)
btcpayserver/btcpayserver: BTCPayServer.Tests/README.md

.\docker-bitcoin-generate.ps1 3
```

### Using Polar to test Lightning payments

> -Install and run [Polar](https://lightningpolar.com/). Setup a small network of nodes.

Reviewer: ```suggestion
- Install and run [Polar](https://lightningpolar.com/). Setup a small network of nodes.
```

> -Goto your Store's General Settings and enable Lightning

Reviewer: ```suggestion
- Goto your Store's General Settings and enable Lightning
```

> - Goto your Store's General Settings and enable Lightning

Reviewer 2: `Goto` should be `Go to`

> - Goto your Store's General Settings and enable Lightning

Reviewer 2: Does `Store` need to be capitalized here?

> -Build you connection string using the Connect infomation in the Polar app

Reviewer: ```suggestion
- Build you connection string using the Connect information in the Polar app
```

> - Build you connection string using the Connect information in the Polar app

Reviewer 2: `Build you` should probably be `Build your` here?

LND Connection string example: 
type=lnd-rest;server=https://127.0.0.1:8084/;macaroonfilepath="local path to admin.macaroon on your computer, without these quotes";allowinsecure=true

> Now you can create a lightning invoice on BTCPayserver regtest and make payment through polar

Reviewer 2: `polar` should be capitalized. Also missing a period at the end of the sentence.
Also, `and make payment` should be `and make a payment`.

> PLEASE NOTE: 

Reviewer 2: I'm not sure what this `PLEASE NOTE:` is for here.

Author: Was going to add a note about a break on exception. Will add and fix the recommendations.

.\docker-bitcoin-generate.ps1 3
```

### Using Polar to test Lightning payments

- Install and run [Polar](https://lightningpolar.com/). Setup a small network of nodes.
- Go to your store's General Settings and enable Lightning.
- Build your connection string using the Connect information in the Polar app.

LND Connection string example: 
type=lnd-rest;server=https://127.0.0.1:8084/;macaroonfilepath="local path to admin.macaroon on your computer, without these quotes";allowinsecure=true

> Now you can create a lightning invoice on BTCPayserver regtest and make a payment through polar.

Reviewer 3: ```suggestion
Now you can create a lightning invoice on BTCPay Server regtest and make a payment through Polar.
```

PLEASE NOTE: You may get an exception break in Visual Studio. You must quickly click "Continue" in VS so the invoice is generated.

> Or, uncheck the box that says, "Break when this exceptiontype is thrown".

Reviewer 2: `exceptiontype` looks like there is a space missing?

Author: Yes. Will fix.
